src/ducks/Watchlists/Widgets/TopBar/TopBar.js

```diff
@@ -76,37 +133,85 @@ const TopBar = ({
       )}
     >
       <div className={styles.info}>
-        <Author
-          avatarUrl={avatarUrl}
-          userName={userName}
-          fullName={fullName}
-          isLoggedIn={isLoggedIn}
-          isCurrentUser={isAuthor}
-          userId={user.id}
-          currentUserId={currentUser && currentUser.id}
-          watchlists={watchlists}
-          addressesWatchlists={addressesWatchlists}
-          type={type}
-        />
-        <div className={styles.leftDivider} />
-        <Title
+        <CreationInfo
+          id={id}
+          type={
+            type === BLOCKCHAIN_ADDRESS
+              ? CreationType.AddressWatchlist
+              : CreationType.Watchlist
+          }
           title={title}
-          description={description}
-          type={type}
-          refetchAssets={refetchAssets}
-          entity={entity}
-          isLoggedIn={isLoggedIn}
-          isCurrentUser={isAuthor}
-        />
-        <CommentActions
-          entity={entity}
-          isLoggedIn={isLoggedIn}
-          isCommentsOpen={isCommentsOpen}
-          setIsCommentsOpen={setIsCommentsOpen}
+          user={{
+            id: +userId,
+            username: userName
+          }}
           currentUser={currentUser}
-          closeFilter={closeFilter}
+          onEditClick={() => setIsEditFormOpened(prev => !prev)}
+          comments={{
+            count: commentsCount,
+            active: isCommentsOpen,
+            onClick: () => {
+              setIsCommentsOpen(prev => !prev)
+              closeFilter()
+            }
+          }}
+          votes={{
+            totalVotes: votes.totalVotes,
+            userVotes: votes.userVotes
+          }}
+          onVote={onVote}
+        />
```

Comment:
[Info tooltip](https://github.com/santiment/san-studio/blob/79e467d4088b72b170b94fc83393ff7a76161adc/src/Header/Layout.svelte#L74-L76) content is missing